CONTRIBUTING.md

```diff
@@ -72,27 +72,16 @@ The `render-notebooks.R` script adds a `bibliography:` specification in the `.Rm
 Default is the `references.bib` script at the top of the repository.  
 - `--html`: Default is to save the output `.html` file the same name as the input `.Rmd` file. This option allows you to specify an output file name. Default is used by snakemake.
 
-## General guidelines for analyses notebooks
 
-Each analysis `.Rmd` notebook needs to be entirely self-contained so that a user can download the `.Rmd` file and have all the necessary steps and information to complete the example analysis.
+## Adding a new analysis
 
-**.Rmd Header**
+Copy, paste and rename the `template/template_example.Rmd` file to start a new analysis and the `template/screenshots` folder.
+Search for `<<` or `>>` and replace those with the pertinent information.
+The introductory info in this template file helps toward our goal of these analyses notebooks being self-contained.
 
-Each analysis `.Rmd` should have this header below:
-This will automatically number the sections, _so no manual numbering should be used_.
+### General guidelines for analyses notebooks
 
-```
----
-title: "Name of the analysis - Microarray/RNA-seq/Advanced"
-author: "CCDL for ALSF"
-date:"`r format(Sys.time(), '%B %Y')`"
-output:   
-  html_notebook:
-    toc: true
-    toc_float: true
-    number_sections: true
----
-```
+Each analysis `.Rmd` notebook needs to be entirely self-contained so that a user can download the `.Rmd` file and have all the necessary steps and information to complete the example analysis.
 
 **Inputs:**  
```

Comment:
Is there a reason not to make these headers as well? I generally like using headers for semantic reasons, and we can still exclude them from the TOC below a certain threshold.
```suggestion
#### Inputs:
```